### PR TITLE
Endless OS: Update to EOS5

### PR DESCRIPTION
**Pull request by cassidyjames**
## Page Metadata Updates

If you're just updating information/metadata on a page make sure you do the following:.

- [x] update the screenshot file to the latest version (if applicable) and please optimize the file size
- [x] update or add the `updated:` field to the post's frontmatter with the date in `YYYY-MM-DD` format
- [x] update or add the `updater:` field to the post's frontmatter with your name in quotations `"Your Name"`
- [x] rename the post's filename with the new date, e.g `YYYY-MM-DD-name`

### CODE REVIEW

**Comment by cassidyjames** [comment]
@waldyrious is this repo the correct place to contribute updates?

**Comment by waldyrious** [comment]
Thanks for the update, @cassidyjames! For the time being, yeah, I intend to maintain this repository and integrate community contributions. Depending on @snwh's response on #2 we can later upstream these changes to the original repo, or make the explicit decision to keep this indefinitely as a fork (in which case we might e.g. set up an organization so that the control of the project, publication URL, etc. isn't directly dependent on my account).

File: _posts/2023-10-10-endless-os.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+layout: distro
+author: "Sam Hewitt"
+updater: "cassidyjames"
+title: "Endless OS"
+date: "2013-09-14"
```
Comment (waldyrious) [inline _posts/2023-10-10-endless-os.md]: Why did you change this from the previous value of 2018-06-01?

File: _posts/2023-10-10-endless-os.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+updated: "2023-10-10"
+logo: "endlessos"
+image: "endlessos-5.png"
```
Comment (waldyrious) [inline _posts/2023-10-10-endless-os.md:9]: Shouldn't we delete the previous `endlessos-3.4.png` file from the repo, since it's no longer used by any page (AFAICT)?

File: _posts/2023-10-10-endless-os.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  version: "5.0.6"
+  based-on: ["Debian"]
+  desktops: [GNOME]
+  install: graphical
+  developers:
+    - name: Endless OS Foundation
+      url: https://endlessos.org/
+---
+
+Endless OS is a beginner-friendly distribution with an interface designed to "feel natural and intuitive", taking cues from mobile operating systems and presenting apps as a grid of icons right on main screen. It's designed so that no Internet is required, but also works well on Internet-connected computers.
```
Comment (waldyrious) [inline _posts/2023-10-10-endless-os.md]: While we're at it :)

```suggestion
Endless OS is a beginner-friendly distribution with an interface designed to "feel natural and intuitive", taking cues from mobile operating systems and presenting apps as a grid of icons right on the main screen. It's designed so that no Internet is required, but also works well on Internet-connected computers.
```

File: _posts/2023-10-10-endless-os.md (hunk 1 above)
Comment (cassidyjames) [reply _posts/2023-10-10-endless-os.md]: That's an excellent question; I meant `2023-09-14`, as that was the latest release 😄 

```suggestion
date: "2023-09-14"
```

but I think I misunderstood the directions in the PR template

File: _posts/2023-10-10-endless-os.md (hunk 2 above)
Comment (cassidyjames) [reply _posts/2023-10-10-endless-os.md:9]: Can do, but it looked like old assets were still around for other releases? I'm indifferent, though.

File: _posts/2023-10-10-endless-os.md (hunk 1 above)
Comment (waldyrious) [reply _posts/2023-10-10-endless-os.md]: Ah, yes, those instructions (and CONTRIBUTING.md too) could be clearer. This date is supposed to be the original date the distro was added to the site. It may help to think of these metadata fields as being related to the post, not the distro. I will change those files to make this more obvious.

We could add a new `latest_release` subfield under `notes` (since those are indeed related to the distro itself), but I'm afraid that would require much more frequent updates to remain correct, so it's probably best to avoid such time-sensitive data.

As for this PR, yes, let's restore the `2018-06-01` date.

```suggestion
date: "2018-06-01"
```

File: _posts/2023-10-10-endless-os.md (hunk 2 above)
Comment (waldyrious) [reply _posts/2023-10-10-endless-os.md:9]: Oh, nice catch! I hadn't noticed that. I suppose it's nice to keep the old screenshots around so we can compare a distro's evolution over time. Let's keep them, then.

**Comment by cassidyjames** [comment]
@waldyrious alright, I went over the suggestions and I think this is back into a good place for review. 😄

**Comment by waldyrious** [comment]
Oh, you were faster than me! I was making the changes locally and cleaning up the commits, but didn't expect you to see my comments before I had submitted the changes. Sorry about that.

Would you mind if I force-push the branch with the cleaned-up commits? In particular, I split the updating of the description into a separate commit so that git can detect that the same file was edited and renamed rather than one file deleted and another added. I kept your original commit's metadata and your authorship for the second commit.

**Comment by cassidyjames** [comment]
@waldyrious works for me :)

**Comment by waldyrious** [comment]
Great! I've now pushed the updates. The [content is the same](https://github.com/waldyrious/computefreely/compare/ed6e2c276fbcf96947029f062399716be98b842d..5f4af2669130fe9dd8ff01b2d28c0dfc53e06655) but the commits (27711159ac99e5c43298ec08241e5cff8b13ac88 and 5f4af2669130fe9dd8ff01b2d28c0dfc53e06655) are more readable now. I'll go ahead and merge. Thanks again for the PR!

File: _posts/2023-10-10-endless-os.md (hunk 1 above)
Comment (waldyrious) [reply _posts/2023-10-10-endless-os.md]: Update: I've proposed improvements to the contribution documentation in #7.
